.gitignore

```diff
@@ -19,6 +19,9 @@ zeppelin-server/derby.log
 
 lens/lens-cli-hist.log
 
+# bin file
+bin/derby.log
```

Comment:
It looks related where we can run the script actually. :-) @cloverhearts Can you handle all of `derby.log` without location?